api/specs.proto

```diff
@@ -393,6 +393,9 @@ message SecretSpec {
 	// The currently recognized values are:
 	// - golang: Go templating
 	Driver templating = 3;
+
+	// Driver is the the secret driver that is used to store the specified secret
+	Driver driver = 4 [(gogoproto.nullable) = false];
```

Comment:
Are we sure this shouldn't be nullable?
I see a lot of checks for `if spec.Driver.Name != "" {}`